src/ts/fileDescriptorTSServices.ts

```diff
@@ -25,6 +41,10 @@ export function printFileDescriptorTSServices(fileDescriptor: FileDescriptorProt
 
   fileDescriptor.getDependencyList().forEach((dependency: string) => {
     const pseudoNamespace = filePathToPseudoNamespace(dependency);
+    // Don't output unused variables as that causes TS6133 error
+    if (!isUsed(fileDescriptor, pseudoNamespace, exportMap)) {
```

Comment:
This can be moved into the functional iterator. We can write something like 
```
fileDescriptor.getDependencyList()
   .filter(dependency => isUsed(fileDescriptor, pseudoNamespace, exportMap))
   .forEach((dependency: string) => {...})
```

Reply:
Updated.